xpdacq/simulation.py

```diff
@@ -51,53 +51,58 @@ def __init__(self, frame_acq_time=0.1, acquire=1):
 
 
 # define simulated PE1C
-class SimulatedPE1C(be.ReaderWithFileStore):
+class SimulatedPE1C(be.ReaderWithRegistry):
     """Subclass the bluesky plain detector examples ('Reader');
 
     also add realistic attributes.
     """
 
-    def __init__(self, name, read_fields, fs):
+    def __init__(self, name, read_fields, reg):
         self.images_per_set = PutGet()
         self.number_of_sets = PutGet()
         self.cam = SimulatedCam()
         self._staged = False
-        super().__init__(name, read_fields, fs=fs)
+        super().__init__(name, read_fields, reg=reg)
         self.ready = True  # work around a hack in Reader
 
 
 def build_pymongo_backed_broker():
-    """Provide a function level scoped MDS instance talking to
+    '''Provide a function level scoped MDS instance talking to
```

Comment:
Agree. I was planning to fix everything in ``xpdSim`` where is the place we will import simulation objects eventually.